.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,25 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+error=$( twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee /dev/tty ; echo "status: ${PIPESTATUS[0]}" )
+if [ "${error: -9}" == "status: 0" ]; then
+   echo "Uploaded wheel to pypi"
+else
+    if [[ $error == *"500"* ]]; then
+        echo "Server error 500"
+        exit
+    fi
+fi
+
+error=$( twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee /dev/tty ; echo "status: ${PIPESTATUS[0]}" )
+if [ "${error: -9}" == "status: 0" ]; then
+   echo "Uploaded tar to pypi"
+else
+    if [[ $error == *"500"* ]]; then
+        echo "Server error 500"
+        exit
+    fi
+fi
```

Comment:
All I can do to make it shorter is to change `echo "status: ${PIPESTATUS[0]}"` to `echo "st: ${PIPESTATUS[0]}"` or something similar.
This is the only way I could store the output as a variable, print the error and store the pipe status at the same time.

I can add comments in the code explaining what each line is doing to make it more understandable.
Should I do that?